torch/onnx/symbolic_registry.py

```diff
@@ -26,6 +26,8 @@ def register_version(domain, version):
 def register_ops_helper(domain, version, iter_version):
     version_ops = get_ops_in_version(iter_version)
     for op in version_ops:
+        if op[0] == '_len':
```

Comment:
why do we do this?

Reply:
(nit : if there are more than one similar case we should eventually add them in a separate function that we call here)

Reply:
Agreed.. So far this is the only case that i could find, so I'm treating it as a special case handling.